low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -1006,6 +1074,55 @@ class SSPLHealthView(object):
                         f"of '{interface}'. {str(err)}")
         return phy_link_state[carrier_indicator]
 
+    def _sanitize_dbus_value(self, value):
+        """Convert certain DBus type combinations so that they are easier to read"""
+        if isinstance(value, Array) and value.signature == "ay":
+            try:
+                return self._decode_ay(value)
+            except:
+                # Try an array of arrays; 'aay' which is the symlinks
+                return list(map(self._decode_ay, value or ()))
+        elif isinstance(value, Array) and value.signature == "y":
+            return bytearray(value).rstrip(bytearray((0,))).decode('utf-8')
+        else:
+            return value
+
+    def _decode_ay(self, value):
+        """Convert binary blob from DBus queries to strings"""
+        if len(value) == 0 or \
+            value is None:
+            return ''
+        elif isinstance(value, str):
+            return value
+        elif isinstance(value, bytes):
+            return value.decode('utf-8')
+        else:
+            return bytearray(value).rstrip(bytearray((0,))).decode('utf-8')
+
+    def _is_physical_drive(self, interfaces_and_property):
```

Comment:
Where is this used?

Reply:
We haven't used this in the above code. but we may need when we need physical/jbod disk information in health view. like we have in the main code. It is just 2,3 lines of code.

Reply:
ok